### ISSUE TITLE
Multi-values and All values variable values handling

### ISSUE THREAD

**Issue by fabiomx** [michelin/snowflake-grafana-datasource#84]
Hello,

I encountered an issue when using dashboard variables with the Multi-values and/or All values option enabled. 

The generated query contains curly brackets that prevent it from working correctly, like this:

```
AND FIELD_NAME IN ('{VALUE1,VALUE2}')
```

The expected query format is:

```
AND FIELD_NAME IN ('VALUE1','VALUE2')
```

**Comment by fabiomx** [comment]
That was quick! Thanks! :)

**Comment by devnied** [comment]
I implemented a custom solution in version 1.7.0, but we need to rely on the Grafana implementation. Therefore, please use version 1.7.1.
I have updated the [documentation](https://github.com/michelin/snowflake-grafana-datasource?tab=readme-ov-file#query-variables), you must use ${variable:sqlstring} to format your variable instead of just ${variable}.
